docs/beginner/tutorial3-pipeline/README.md

```diff
@@ -4,25 +4,25 @@
 If you're familiar with OpenGL, you may remember using shader programs. You can think of a pipeline as a more robust version of that. A pipeline describes all the actions the gpu will preform when acting on a set of data. In this section, we will be creating a `RenderPipeline` specifically.
 
 ## Wait shaders?
-Shader's are mini programs that you send to the gpu to perform operations on your data. There are 3 main types of shader: vertex, fragment, and compute. There are others such as geometry shaders, but they're more of an advanced topic. For now we're just going to use vertex, and fragment shaders.
+Shaders are mini programs that you send to the gpu to perform operations on your data. There are 3 main types of shader: vertex, fragment, and compute. There are others such as geometry shaders, but they're more of an advanced topic. For now we're just going to use vertex, and fragment shaders.
 
 ## Vertex, fragment.. what are those?
 A vertex is a point in 3d space (can also be 2d). These vertices are then bundle in groups of 2s to form lines and/or 3s to form triangles. 
 
 <img src="./tutorial3-pipeline-vertices.png" />
 
-Most modern rendering uses triangles to make simple shapes such as a cube, and complex shapes such as person.
+Most modern rendering uses triangles to make simple shapes such as cube, and complex shapes such as person.
 
 <!-- Todo: Find/make an image to put here -->
 
-We use a vertex shader to manipulate a list of vertices, in order to transform the shape to look they way we want it.
+We use a vertex shader to manipulate a list of vertices, in order to transform the shape to look the way we want it.
 
-You can think of a fragment as the beginnings of a pixel in the resulting image. Each fragment has a color that will be copied to it's corresponding pixel. The fragment shader decides what color the fragment will be.
```

Comment:
"a fragment" would be correct here as well as "the beginnings". A pixel can have multiple fragments. The number of fragments depends on the number of samples when multisampling